assets/js/blocks/cart-checkout/cart/full-cart/style.scss

```diff
@@ -253,7 +254,44 @@ table.wc-block-cart-items {
 	}
 	.wc-block-cart__sidebar .components-card {
 		@include placeholder();
+		@include force-content();
+	}
+}
+.wc-block-cart--skeleton {
+	display: none;
+
+	th span,
+	h2 span {
+		@include placeholder();
+		@include force-content();
+		min-width: 84px;
+		display: inline-block;
+	}
+	.wc-block-cart-item__image div,
+	.wc-block-quantity-selector {
+		@include placeholder();
 	}
+	.wc-block-cart-item__product,
+	.wc-block-cart-item__total {
+		div {
+			@include placeholder();
+			@include force-content();
+			min-width: 84px;
+			display: inline-block;
+		}
+		div + div {
+			margin-top: 0.25em;
+			min-width: 8em;
+		}
+	}
+	.wc-block-cart__sidebar .components-card {
+		@include placeholder();
+		@include force-content();
+		min-height: 400px;
+	}
+}
+.is-loading + .wc-block-cart--skeleton {
+	display: flex;
```

Comment:
I'm curious, couldn't we just reuse the styles under `.wc-block-cart--is-loading`? If the markup is the same, the selectors should be reusable (you would only need to add the `.wc-block-cart--skeleton` class).

Reply:
The markup isn't identical. Placeholder has some real content, but this uses empty divs. We can try consolidating this.